src/pages/workspace/distanceRates/CreateDistanceRatePage.tsx

```diff
@@ -46,6 +46,10 @@ function CreateDistanceRatePage({route}: CreateDistanceRatePageProps) {
     );
 
     const submit = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.POLICY_CREATE_DISTANCE_RATE_FORM>) => {
+        if (!customUnitID) {
+            return;
+        }
```

Comment:
Can we please add a comment here saying that a blocking view is shown if the customUnitID is unset, so the submit function will never run in that case?